package.json

```diff
@@ -33,8 +33,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*.test.js",
-    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --runInBand --colors ./src/integration/pages/**/*.test.js",
+    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --no-cache --runInBand --colors ./src/integration/pages/**/*",
```

Comment:
Might be safer to remove `--no-cache` (apologies I know I requested to add this to the command initially). None of the other commands which use jest use this param, and the docs state that: _Note: the cache should only be disabled if you are experiencing caching related problems. On average, disabling the cache makes Jest at least two times slower_ - https://jestjs.io/docs/en/cli.html#--cache